run-tests.php

```diff
@@ -1505,9 +1464,15 @@ function run_all_tests_parallel($test_files, $env, $redir_tested) {
 	echo "\n";
 
 	$rawMessageBuffers = [];
+	$testsInProgress = 0;
+
+	// Map from conflict key to worker ID.
+	$activeConflicts = [];
+	// Tests waiting due to conflicts. Map from conflict key to array.
+	$waitingTests = [];
 
 escape:
-	while ($testDirsToGo || ($testDirsInProgress > 0)) {
+	while ($test_files || $testsInProgress > 0) {
```

Comment:
Why test_files and not testFiles?

Reply:
`$test_files` is an existing variable. Would have to touch a lot of code (which all seems to use underscore style) to rename it.